app/backend/src/tests/test_bg_jobs.py

```diff
@@ -649,6 +651,105 @@ def test_process_send_onboarding_emails(db):
         )
 
 
+def test_process_send_reference_reminders(db):
+    # need to test:
+    # case 1: bidirectional (no emails)
+    # case 2: host left ref (surfer needs an email)
+    # case 3: surfer left ref (host needs an email)
+    # case 4: neither left ref (host & surfer need an email)
```

Comment:
Seems to me this should be broken up into 4 tests, each of which would tell us if its one case is broken.

Reply:
Part of the test is to make sure that it does it fine when there's a set of emails to send and not just one-off.

Anyway, it's faster to combine them into one unit test, and not too difficult if it breaks to figure out which one broke

Reply:
Theoretically should be 5 tests then, but OK